.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: npm ci
+      - run: npm test
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+      - run: |
+          cd ${{ env.GITHUB_WORKSPACE }}/contracts
+          cp ../README.md README.md
+          npm ci
+          npm publish
```

Comment:
We don't need public option here?
```
npm publish --access public
```

Otherwise, github action module can be also good.
```yaml
    - uses: JS-DevTools/npm-publish@v1
      with:
        token: ${{ secrets.NPM_TOKEN }}
        access: public
```
https://github.com/klaytn/kaikas-web3-provider/blob/main/.github/workflows/publish.yml#L15